### PR TITLE
chore: [CDS-99648]: Add providers icon for oauth for on-prem

### PR DESCRIPTION
**Pull request by sonaligoyal-harness**
You can use the following comments to re-trigger PR Checks

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- StyleLint: `retrigger stylelint`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- ImageSnapshot `retrigger ImageSnapshot`

### CODE REVIEW

**Comment by bot2-harness** [comment]
Image snapshots failed please check https://console.cloud.google.com/storage/browser/ui-team-bucket/uicore-2649

File: packages/icons/src/providers.svg
Comment (shivanand-harness) [inline packages/icons/src/providers.svg:1]: remove height and width properties and check if we can compress the svg image size using this https://vecta.io/nano

**Review by shivanand-harness — CHANGES_REQUESTED** [review]

File: packages/icons/src/providers.svg
Comment (sonaligoyal-harness) [reply packages/icons/src/providers.svg:1]: I compressed it using yarn svgo, will check using the above one

**Comment by bot2-harness** [comment]
Image snapshots failed please check https://console.cloud.google.com/storage/browser/ui-team-bucket/uicore-2651

**Pull request closed without merging**